src/kernel/log.zig

```diff
@@ -9,7 +9,7 @@ pub const Level = enum {
 };
 
 fn logCallback(context: void, str: []const u8) anyerror!void {
-    serial.writeString(str, serial.Port.COM1);
+    serial.writeBytes(str, serial.Port.COM1);
```

Comment:
What was wrong with writeString?

Reply:
IMO it makes its use more clear and less specific, since the function is *actually* used to send bytes rather than specifically a string.